src/main/scala/com/databricks/spark/csv/package.scala

```diff
@@ -32,6 +32,7 @@ package object csv {
                 delimiter: Char = ',',
                 quote: Char = '"',
                 escape: Character = null,
+                comment: Character = null,
```

Comment:
We are getting a style error here. That is because we are exceeding 11 parameters. I am fixing that in a separate PR.